charts/sf-archival/charts/query-controller/values.yaml

```diff
@@ -36,6 +36,12 @@ db:
   queryexecutioncontroller:
     max_idle_connections: 1
     max_open_connections: 2
+parallelQuery:
+  queryController:
+    queryParallelismFactor: 5
+    steppedParallelism: false
```

Comment:
As mentioned if default value of  steppedParallelism is false then we need not give here.

Reply:
as discussed we will not expose the steppedParallelism value in the overriding values file we use while upgrading